docs/control-structures.rst

```diff
@@ -122,7 +122,7 @@ A contract can also have a default function, which is executed on a call to the
 
 This function is always named ``__default__``. It must be annotated with ``@external``. It cannot expect any input arguments.
 
-If the function is annotated as ``@payable``, this function is executed whenever the contract is sent Ether (without data). This is why the default function cannot accept arguments - it is a design decision of Ethereum to make no differentiation between sending ether to a contract or a user address.
+If the function is annotated as ``@payable``, this function is executed whenever the contract is sent Ether (without data). This is why the default function cannot accept arguments — it is a design decision of Ethereum to make no differentiation between sending Ether to a contract or a user address.
```

Comment:
does the different in hyphen show up in the final output?

Reply:
It does so when I preview it in VSCode.